packages/pages/src/dev/server/middleware/indexPage.ts

```diff
@@ -286,3 +304,32 @@ const createFunctionsTable = (
   }
   return indexPageHtml;
 };
+
+const createYextAccountUrl = (accountId: string, universe: string) => {
```

Comment:
Can you move this function to its own util as well, called getYextUrlForPartition that returns e.g. `app.eu.yext.com`. I think it will be used in other places as well. Then you can do the `https://${getYexturlForPartition…/s/${accountId}/` on line 58.

Reply:
Add tests too